plutus-ledger-api/test-plugin/Spec/Data/Value/9.6/Long.stat.golden

```diff
@@ -0,0 +1,15 @@
+({cpu: 6727061187 | mem: 12083358})
+({cpu: 10186791 | mem: 31548})
+({cpu: 4747794921 | mem: 8436358})
+({cpu: 10186791 | mem: 31548})
+({cpu: 4747794921 | mem: 8436358})
+({cpu: 10186791 | mem: 31548})
+({cpu: 2768528655 | mem: 4789358})
+({cpu: 10186791 | mem: 31548})
+({cpu: 4747794921 | mem: 8436358})
+({cpu: 10186791 | mem: 31548})
+({cpu: 2768528655 | mem: 4789358})
+({cpu: 10186791 | mem: 31548})
+({cpu: 2768528655 | mem: 4789358})
+({cpu: 10186791 | mem: 31548})
+({cpu: 789262389 | mem: 1142358})
```

Comment:
Non-data version:
```
({cpu: 3007736773 | mem: 11157042})
({cpu: 712988980 | mem: 3097504})
({cpu: 2101129911 | mem: 7793754})
({cpu: 587638980 | mem: 2552504})
({cpu: 2101129911 | mem: 7793754})
({cpu: 462012980 | mem: 2006304})
({cpu: 1194523049 | mem: 4430466})
({cpu: 462150980 | mem: 2006904})
({cpu: 2101129911 | mem: 7793754})
({cpu: 462012980 | mem: 2006304})
({cpu: 1194523049 | mem: 4430466})
({cpu: 336662980 | mem: 1461304})
({cpu: 1194523049 | mem: 4430466})
({cpu: 211036980 | mem: 915104})
({cpu: 287916187 | mem: 1067178})
```

Reply:
Funny, I thought these tests were complete garbage (when I wrote them I had the wrong assumptions about what `Value` was, so the tests aren't testing anything sensible really), but here they kinda show us that the new version computes more than necessary? All these old `mem: 31548` vs new `mem: 1461304` etc.

Reply:
Actually, those are the tests which show improvement! Some of them are worse, yes, but I'm pretty sure it's because of `geq` etc. which I will come back to in a future PR.